src/components/travel-route/indoor-directions-steps.component.tsx

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import {
+  View,
+  TouchableOpacity,
+  Text,
+  ScrollView,
+  StyleSheet,
+  Animated,
+} from "react-native";
+import SlidingUpPanel from "rn-sliding-up-panel";
+import { AntDesign, Feather } from "@expo/vector-icons";
+import { FloorPath } from "../../types/main";
+import {
+  CONCORDIA_RED,
+  screenHeight,
+  WHITE_BACKGROUND_COLOR,
+} from "../../styles";
+import IndoorPathPlanningService from "../../services/indoor-path-planning.service";
+
+interface IProps {
+  floorPaths: FloorPath[];
+  onCloseTravelSteps: () => void;
+}
+
+interface IState {
+  allowDragging: boolean;
+}
+
+/**
+ * A sliding panel that shows the steps to the destination
+ */
+class IndoorDirectionSteps extends React.Component<IProps, IState> {
+  _panel: SlidingUpPanel;
+
+  _draggedValue: Animated.Value;
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      allowDragging: true,
+    };
+  }
```

Comment:
Oh man been a while since I've seen a react class. Not a blocker or change request, just a comment :D